lib/portlayer/storage/vsphere/datastore_test.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vsphere
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/vmware/vic/pkg/vsphere/session"
+	"github.com/vmware/vic/pkg/vsphere/tasks"
+	"github.com/vmware/vic/pkg/vsphere/test/env"
+	"golang.org/x/net/context"
+)
+
+func Session(ctx context.Context, t *testing.T) *session.Session {
```

Comment:
I tried using the `test` package but none of the default inventory paths (i.e datacenter name, network name, etc.) mapped to the environment I was testing against (VC with non standard cluster, datacenter, and datastore names).  And apparently calling `Create()` validates all of those entries so just making my own Session seemed logical.  I think going forward we can replace all the inventory paths in `test.Session` with params extracted from environment variables or some default.  This was a quick hack.